letsdrill.rb

```diff
@@ -1,24 +1,40 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  percentageGrade = integer.to_i
+
+  puts case percentageGrade
+  when 90..100
+    return "A"
+  when 80..89
+    return "B"
+  when 70..79
+    return "C"
+  when 60..69
+    return "D"
+  when 0..59
+    return "F"
+  else
+    return "Invalid percentage"
```

Comment:
`return` can be omitted in ruby, and standard practice is to omit it unless it's and early-return.